t/schema/Schedule-1.yaml

```diff
@@ -0,0 +1,65 @@
+$id: Schedule-1.yaml
+$schema: http://json-schema.org/draft-04/schema#
+description: TODO
+type: object
+additionalProperties: false
+required:
+# TODO Every file should have a description?
+#- description
```

Comment:
Definitely, it should be required

Reply:
That's weird, I commented it out because I got some files that didn't have a description.
Just tried it out again and now all files pass =)

Will remove the comment...

Reply:
I guess not every schedule has it atm, and on top we have sometimes multiple schedules, where we duplicate description, but will use merge keys feature to avoid it in future. So I would not force description to be defined here, as there are multiple approaches in different teams. Some rely on openQA test suites, for example.

Reply:
Hm, I removed a description from a file and tests still pass. Somethings wrong with the schema...

Reply:
Stupid me.

I made a mistake in test_yaml_valid.
So we have around files without a description field, so I suggest I keep that commented out for now.
Also there are 5 files that have a `description:` without a value.

Reply:
@rwx788 do you have an example on where the description is duplicated?. My point on description being obligatory would be: There has to be a reason for that particular schedule to exist. But I guess that could be enabled later on. 

PS: @perlpunk FSM is using it's tentalcess to mess with JSON and YAML altogether! RUN!

Reply:
@foursixnine For example https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/schedule/yast/lvm_raid1 as instead putting complex conditions in the schedule itself we sometimes create separate file for some exotic backend or setup. In this case we could have imported common description for all of them, as well as some settings.

Reply:
Point taken :)